public/app/plugins/datasource/azuremonitor/components/LogsQueryEditor/TimeManagement.tsx

```diff
@@ -80,7 +80,6 @@ export function TimeManagement({ query, onQueryChange: onChange, schema }: Azure
 
   useEffect(() => {
     if (query.azureLogAnalytics?.basicLogsQuery) {
-      onChange(setDashboardTime(query, 'dashboard'));
```

Comment:
I think you still need this line (it's causing a failing test)

Reply:
Having this here was causing an infinite loop amongst the dependency updates so I have this handled by the `LogsManagement` component, I have updated the tests both here and for `LogsManagement` to make sure this is accounted for.